vkcs/compute/resource_instance_test.go

```diff
@@ -760,6 +760,75 @@ func TestResourceInstanceBlockDevicesV2(t *testing.T) {
 	}
 }
 
+func TestResourceComputeInstanceCloudMonitoringBasic(t *testing.T) {
```

Comment:
Test suffixes should be added as it is commonly done for acceptance tests.

Reply:
fix test name